### PR TITLE
feat: LT-1328 - Open licence refactor

### PR DESCRIPTION
**Pull request by jfaracik**
* Split standard and open case pages - now one page each, as opposed to one combined page
* Little styling changes here and there - default font and font size changed, red accent colour removed
* Removed register_business app - moved into organisations

### CODE REVIEW

File: assets/css/overrides/_table.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.govuk-table {
```
Comment (a-bek) [inline assets/css/overrides/_table.scss]: Do we want to override? This affects all tables in lite?

File: cases/case_types.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+STANDARD_LICENCE = "standard_licence"
```
Comment (a-bek) [inline cases/case_types.py]: I'd keep the old file constants and turn these into a CaseType Enum?

File: assets/css/overrides/_table.scss (hunk 1 above)
Comment (jfaracik) [reply assets/css/overrides/_table.scss]: Yep it does - overrides all tables on internal LITE. It sets the header to a set height which makes it easier for alignment, such as aligning checkboxes in the header.

File: .circleci/config.yml [hunk 3]
```diff
@@ -72,7 +72,7 @@ jobs:
 
       - run:
           name: check code formatting
-          command: pipenv run black . --line-length 120 --check --diff
```
Comment (andytaylorkainos) [inline .circleci/config.yml]: why did you get rid of this?

File: cases/case_types.py (hunk 2 above)
Comment (jfaracik) [reply cases/case_types.py]: Seems like a good shout, done.

File: organisations/views.py [hunk 4]
```diff
@@ -62,3 +62,48 @@ def get(self, request, **kwargs):
...
+    def get(self, request, **kwargs):
+        return form_page(request, self.forms.forms[0])
+
+    def post(self, request, **kwargs):
+        response, _ = submit_paged_form(request, self.forms, post_organisations)
+
+        if response:
+            return response
+
+        messages.success(request, "The organisation was created successfully")
```
Comment (andytaylorkainos) [inline organisations/views.py]: Content strings

File: templates/case/includes/supporting-documentation.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<h3 class="govuk-heading-m">Supporting documentation</h3>
+{% if case.application.supporting_documentation %}
+	<table class="govuk-table" id="supporting-documentation">
+		<thead class="govuk-table__head">
+			<tr class="govuk-table__row">
+				<th class="govuk-table__header" scope="col">Name</th>
```
Comment (andytaylorkainos) [inline templates/case/includes/supporting-documentation.html]: Content strings

File: templates/case/standard-licence-case.html [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+		{% for flag in case.flags %}
+			<div class="app-flag" id="{{ flag.name }}">
+				{{ flag.name }}
+			</div>
+		{% endfor %}
+		<a href="{% url 'cases:assign_flags' case.id %}?level=cases&items={{ case.id }}" id="application-edit-case-flags" class="app-flag app-flag--action">
+			{% if case.flags %}
+				{% get_string 'cases.case.edit_case_flags' %}
+			{% else %}
+				Set case flags
```
Comment (andytaylorkainos) [inline templates/case/standard-licence-case.html]: content string

File: templates/case/standard-licence-case.html [hunk 7]
```diff
@@ -0,0 +1,301 @@
...
+			    {% include 'case/views/standard-goods.html' with show_checkboxes=True %}
+			</form>
+		</div>
+
+		<!-- Goods location -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-goods">
+						Goods location
```
Comment (andytaylorkainos) [inline templates/case/standard-licence-case.html]: content string

File: templates/case/standard-licence-case.html [hunk 8]
```diff
@@ -0,0 +1,301 @@
...
+				</table>
+			</div>
+		</div>
+
+		<!-- Parties -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						Entities involved
```
Comment (andytaylorkainos) [inline templates/case/standard-licence-case.html]: content string

File: templates/case/standard-licence-case.html [hunk 9]
```diff
@@ -0,0 +1,301 @@
...
+				</table>
+			</div>
+		</div>
+
+		<!-- Other Documents -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						Supporting documentation
```
Comment (andytaylorkainos) [inline templates/case/standard-licence-case.html]: content string

File: templates/case/standard-licence-case.html [hunk 10]
```diff
@@ -0,0 +1,301 @@
...
+				{% include 'case/includes/other-documents.html' %}
+			</div>
+		</div>
+
+		<!-- Activity -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						Activity
```
Comment (andytaylorkainos) [inline templates/case/standard-licence-case.html]: string

File: templates/case/views/open-destinations.html [hunk 11]
```diff
@@ -74,4 +74,4 @@ <h3 class="govuk-heading-m">Destinations</h3>
 			</div>
 		{% endfor %}
 	</div>
-{% endif %}
+{% endif %}
```
Comment (andytaylorkainos) [inline templates/case/views/open-destinations.html]: new line

File: templates/case/views/open-goods.html [hunk 12]
```diff
@@ -3,9 +3,9 @@ <h3 class="govuk-heading-m">Goods</h3>
 	<thead class="govuk-table__head">
 		<tr class="govuk-table__row">
 			<th class="govuk-table__header" scope="col">Description</th>
-			<th class="govuk-table__header" scope="col">Control List Entry</th>
+			<th class="govuk-table__header" scope="col">Control list entry</th>
 			<th class="govuk-table__header" scope="col">Countries</th>
-			<th class="govuk-table__header">Is Good controlled</th>
+			<th class="govuk-table__header">Controlled</th>
```
Comment (andytaylorkainos) [inline templates/case/views/open-goods.html]: all of these are strings

File: templates/organisations/organisation.html [hunk 13]
```diff
@@ -89,7 +89,23 @@ <h2 class="govuk-heading-l">{{ title }}</h2>
...
+				{{ flag.name }}
+			</div>
+		{% endfor %}
+		{% if not hide_creation_link %}
+			<a id="link-organisation-flags" class="app-flag app-flag--action" href="{% url 'flags:assign_flags' %}?level=organisations&items={{ organisation.id }}">
+				{% if organisation.flags %}
+					Edit organisation flags
+				{% else %}
+					Set a flag on this organisation
+				{% endif %}
```
Comment (andytaylorkainos) [inline templates/organisations/organisation.html]: strings for these too

File: ui_automation_tests/docker-entry.sh [hunk 14]
```diff
@@ -1,11 +1,17 @@
...
+else
+    py.test -k "$TESTS_TO_RUN" --ignore=core --alluredir=ui_automation_tests/allure-results &
+fi
 pid="$!"
 
 # trap process id to stop script using Control+C
 trap "echo '=== Stopping PID $pid ==='; kill -SIGTERM $pid" SIGINT SIGTERM
 
-wait $pid
+wait $pid
```
Comment (andytaylorkainos) [inline ui_automation_tests/docker-entry.sh]: new line

File: .circleci/config.yml (hunk 3 above)
Comment (jfaracik) [reply .circleci/config.yml]: It reads it from `pyproject.toml`

File: assets/css/styles.scss [hunk 15]
```diff
@@ -1,202 +1,14 @@
...
-
-.lite-tabs__tab {
-	text-decoration: none;
-}
-
-.app-letter-preview__page {
-	font-family: $govuk-font-family;
-	font-size: 18px;
-}
+@import "random"  // Temporary file for odd bits of SCSS
```
Comment (markm5A8Z9F43913A) [inline assets/css/styles.scss]: Temporary file... until when? Also, can the name be more descriptive please

File: assets/javascripts/case.js [hunk 16]
```diff
@@ -1,4 +1,5 @@
-function showFlagsModal() {
-	LITECommon.Modal.showModal("Set flags on this case", $("#flags-modal").html(), false);
-	return false;
-};
+// This may get added back in the future
```
Comment (markm5A8Z9F43913A) [inline assets/javascripts/case.js]: How about we get rid of this and add it when we need it

File: cases/views/main.py [hunk 17]
```diff
@@ -309,7 +314,7 @@ def generate_file(result):
 #             'description': original_file_name,
 #             'case': case,
```
Comment (markm5A8Z9F43913A) [inline cases/views/main.py]: Delete this whole section.

File: organisations/urls.py [hunk 18]
```diff
@@ -7,8 +7,9 @@
 urlpatterns = [
     # ex: /organisations/
     path("", views.OrganisationList.as_view(), name="organisations"),
-    # ex: /organisations/43a88949-5db9-4334-b0cc-044e91827451/
     path("<uuid:pk>/", views.OrganisationDetail.as_view(), name="organisation"),
-    # ex: /organisations/hmrc/
     path("hmrc/", views.HMRCList.as_view(), name="hmrc"),
+    # Register
```
Comment (markm5A8Z9F43913A) [inline organisations/urls.py]: The comment doesn't really add value.

File: organisations/views.py [hunk 19]
```diff
@@ -62,3 +62,48 @@ def get(self, request, **kwargs):
...
+    def get(self, request, **kwargs):
+        return form_page(request, self.forms.forms[0])
+
+    def post(self, request, **kwargs):
+        response, _ = submit_paged_form(request, self.forms, post_organisations)
+
+        if response:
+            return response
+
+        messages.success(request, "The HMRC organisation was created successfully")
```
Comment (markm5A8Z9F43913A) [inline organisations/views.py]: strings

File: templates/case/includes/other-documents.html [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+			{% for supporting_documentation in case.application.additional_documents %}
+			<tr class="govuk-table__row">
+				<td class="govuk-table__cell">
+					{{ supporting_documentation.name }}
+				</td>
+				<td class="govuk-table__cell">
+					{{ supporting_documentation.description }}
+				</td>
+				<td class="govuk-table__cell">
+					{% if supporting_documentation.safe == True %}
```
Comment (markm5A8Z9F43913A) [inline templates/case/includes/other-documents.html]: Is it possible at all that the supporting documentation is unsafe?

File: templates/case/includes/goods-types.html [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+						<div class="govuk-checkboxes--small govuk-checkboxes__item">
+							<input class="govuk-checkboxes__input" type="checkbox" name="goods" value="{{ good.id }}" id="{{ good.id }}">
+							<label class="govuk-label govuk-checkboxes__label" for="{{ good.id }}">{{ good.description }}</label>
+						</div>
+					</td>
+					<td class="govuk-table__cell">
+						{% if good.control_code %}
+							<span class="app-country-tag app-country-tag--orange">{{ good.control_code }}</span>
+						{% else %}
+							None
```
Comment (markm5A8Z9F43913A) [inline templates/case/includes/goods-types.html]: Strings

File: templates/case/includes/goods-types.html [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+	<button id="button-edit-goods-flags" value="goods" class="govuk-button" data-module="govuk-button">
+		{% get_string 'cases.case.edit_goods_flags' %}
+	</button>
+	<table class="govuk-table">
+		<thead class="govuk-table__header">
+			<tr>
+				<th class="govuk-table__header" scope="col">Description</th>
+				<th class="govuk-table__header" scope="col">Control list entry</th>
+				<th class="govuk-table__header" scope="col">Is good controlled</th>
+				<th class="govuk-table__header" scope="col">Flags</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/includes/goods-types.html]: Strings

File: templates/case/includes/other-documents.html [hunk 23]
```diff
@@ -0,0 +1,33 @@
+{% if case.application.additional_documents %}
+	<table class="govuk-table" id="supporting-documentation">
+		<thead class="govuk-table__head">
+			<tr class="govuk-table__row">
+				<th class="govuk-table__header" scope="col">Name</th>
+				<th class="govuk-table__header" scope="col">Description</th>
+				<th class="govuk-table__header" scope="col">Document</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/includes/other-documents.html]: Strings

File: templates/case/includes/other-documents.html [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+						{{ supporting_documentation.name }}<br>
+					</a>
+					{% endif %}
+				</td>
+			</tr>
+			{% endfor %}
+		</tbody>
+	</table>
+{% else %}
+	No supporting documentation
```
Comment (markm5A8Z9F43913A) [inline templates/case/includes/other-documents.html]: Strings

File: templates/case/open-licence-case.html [hunk 25]
```diff
@@ -0,0 +1,281 @@
...
+			<li class="govuk-breadcrumbs__list-item" aria-current="page">&nbsp;</li>
+		</ol>
+	</div>
+{% endblock %}
+
+{% block body %}
+	<div class="lite-app-bar" style="margin-top: -20px;">
+		<div class="lite-app-bar__content">
+			<h2 class="govuk-heading-l">
+				{% block title %}ABC-123{% endblock %}
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: What is this?

File: templates/case/open-licence-case.html [hunk 26]
```diff
@@ -0,0 +1,281 @@
...
+			<div class="app-flag" id="{{ flag.name }}">
+				{{ flag.name }}
+			</div>
+		{% endfor %}
+	</div>
+
+	<div class="app-case-board">
+		<div class="govuk-grid-row">
+			<div class="govuk-grid-column-one-quarter">
+				<p class="app-case-board__heading">Organisation</p>
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 27]
```diff
@@ -0,0 +1,281 @@
...
+		{% for flag in case.flags %}
+			<div class="app-flag" id="{{ flag.name }}">
+				{{ flag.name }}
+			</div>
+		{% endfor %}
+		<a href="{% url 'cases:assign_flags' case.id %}?level=cases&items={{ case.id }}" id="application-edit-case-flags" class="app-flag app-flag--action">
+			{% if case.flags %}
+				{% get_string 'cases.case.edit_case_flags' %}
+			{% else %}
+				Set case flags
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 28]
```diff
@@ -0,0 +1,281 @@
...
+		</div>
+	</div>
+
+	<div class="govuk-accordion app-accordion" data-module="govuk-accordion" id="accordion-default">
+		<!-- Goods Types -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+						{{ case.application.goods_types|length }} Goods
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 29]
```diff
@@ -0,0 +1,281 @@
...
+	<script src="{% static 'javascripts/more-actions.js' %}"></script>
+	<script src="{% static 'javascripts/edit-good-flags.js' %}"></script>
+	<script src="{% static 'javascripts/modal.js' %}"></script>
+	<script src="{% static 'javascripts/case.js' %}"></script>
+	<script src="{% static 'javascripts/select-buttons.js' %}"></script>
+
+	<script type="text/javascript">
+		$(".govuk-accordion__section").addClass("govuk-accordion__section--expanded");
+	</script>
+{% endblock %}
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Lots of strings

File: core/builtins/custom_tags.py [hunk 30]
```diff
@@ -192,3 +193,56 @@ def get_end_user(application: dict):
...
+
+@register.filter()
+def default_na(value):
+    """
+    Returns N/A if the parameter given is none
+    """
+    if value:
+        return value
+    else:
+        return mark_safe('<span class="lite-hint">N/A</span>')  # nosec
```
Comment (markm5A8Z9F43913A) [inline core/builtins/custom_tags.py]: Strings

File: assets/css/styles.scss [hunk 31]
```diff
@@ -1,202 +1,14 @@
 // Set defaults
 $lite-transition-speed: 0.3s !default;
 $govuk-page-width: 1200px !default;
-$govuk-brand-colour: #cf102d;
-$govuk-font-family: Arial, Sans-Serif;
+$govuk-root-font-size: 18px !default;
+// $govuk-brand-colour: #cf102d;
+$govuk-font-family: -apple-system, BlinkMacSystemFont, Arial, Sans-Serif;
```
Comment (markm5A8Z9F43913A) [inline assets/css/styles.scss]: Should Apple fonts be on top?

Comment (jfaracik) [reply assets/css/styles.scss]: It's the system font for macOS.

File: ui_automation_tests/pages/application_page.py [hunk 32]
```diff
@@ -33,17 +33,14 @@ class ApplicationPage(BasePage):
     MOVE_CASE_BUTTON = '[href*="move"]'  # CSS
     STATUS = "status"  # ID
     AUDIT_TRAIL_ITEM = ".app-activity__item"  # CSS
-    APPLICATION_SUMMARY_BOARD = ".govuk-summary-list"  # CSS
-    EU_TABLE = "end-user"  # ID
-    UEU_TABLE = "ultimate-end-users"  # ID
-    CONSIGNEE_TABLE = "consignee"  # ID
-    THIRD_PARTIES_TABLE = "third-parties"  # ID
+    APPLICATION_SUMMARY_BOARD = ".app-case-board"  # CSS
+    TABLE_ENTITIES = "table-entities"
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/pages/application_page.py]: Comment the type of selector please

File: ui_automation_tests/pages/application_page.py [hunk 33]
```diff
@@ -33,17 +33,14 @@ class ApplicationPage(BasePage):
...
     GIVE_ADVICE_BUTTON = "button-give-advice"  # ID
     CHECKBOX = '[type="checkbox"]'  # CSS
     DOWNLOAD_GOOD_DOCUMENT = "good_document"  # ID
     DOWNLOAD_END_USER_DOCUMENT = "end_user_document"  # ID
-    DOWNLOAD_ADDITIONAL_DOCUMENT = "additional_document"  # ID
-    ORGANISATION = "applicant_organisation"  # CSS
+    DOWNLOAD_ADDITIONAL_DOCUMENT = "supporting-documentation"  # ID
+    LINK_ORGANISATION_ID = "link-organisation"
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/pages/application_page.py]: Comment the type of selector please

Comment (jfaracik) [reply ui_automation_tests/pages/application_page.py]: Is the name not descriptive enough?

File: ui_automation_tests/pages/organisation_page.py [hunk 34]
```diff
@@ -1,17 +1,18 @@
-from helpers.BasePage import BasePage
+from shared.BasePage import BasePage
 
 
 class OrganisationPage(BasePage):
-    EDIT_ORGANISATION_FLAGS = 'a[href*="/assign-flags/"]'
-    FLAGS_AREA = ".app-flag"
+
+    LINK_ORGANISATION_FLAGS_ID = "link-organisation-flags"
+    FLAGS_AREA_SELECTOR = ".app-flag"
```
Comment (markm5A8Z9F43913A) [inline ui_automation_tests/pages/organisation_page.py]: Comment the type of selector please

File: templates/organisations/index.html [hunk 35]
```diff
@@ -86,7 +86,7 @@ <h2 class="govuk-heading-l">Organisations</h2>
 			</thead>
 			<tbody class="govuk-table__body">
 				{% for organisation in data.results %}
-					<tr class="govuk-table__row" style="vertical-align: top;">
+					<tr class="govuk-table__row">
 						<td class="govuk-table__cell">
 							<a class="govuk-link" href="{% url 'organisations:organisation' organisation.id %}">{{ organisation.name|default:"Untitled" }}</a>
```
Comment (markm5A8Z9F43913A) [inline templates/organisations/index.html]: String

File: templates/case/includes/other-documents.html (hunk 20 above)
Comment (jfaracik) [reply templates/case/includes/other-documents.html]: Presumably it just doesn't show the file if it isn't safe, I didn't write it.

File: assets/css/styles.scss [hunk 36]
```diff
@@ -1,202 +1,17 @@
...
-	text-decoration: none;
-}
-
-.app-letter-preview__page {
-	font-family: $govuk-font-family;
-	font-size: 18px;
-}
+@import "placeholder";
+// Temporary file for odd bits of SCSS
+// Will be cleaned up every story
```
Comment (andytaylorkainos) [inline assets/css/styles.scss]: new line

File: templates/case/open-licence-case.html [hunk 37]
```diff
@@ -0,0 +1,281 @@
...
+				</form>
+			</div>
+		</div>
+
+		<!-- Countries -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-2">
+						{{ case.application.destinations.data|length }} Countries
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 38]
```diff
@@ -0,0 +1,281 @@
...
+			</div>
+			<div id="accordion-default-content-2" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
+				<table class="govuk-table">
+					<thead class="govuk-table__head">
+						<tr class="govuk-table__row">
+							<th scope="col" class="govuk-table__header">Green list</th>
+							<th scope="col" class="govuk-table__header">Grey list</th>
+							<th scope="col" class="govuk-table__header">Red list</th>
+							<th scope="col" class="govuk-table__header">Range 8 referral</th>
+							<th scope="col" class="govuk-table__header">Sanctioned</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 39]
```diff
@@ -0,0 +1,281 @@
...
+				</table>
+			</div>
+		</div>
+
+		<!-- Supporting documentation -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						Supporting documentation
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/views/open-goods.html [hunk 40]
```diff
@@ -3,9 +3,9 @@ <h3 class="govuk-heading-m">Goods</h3>
 	<thead class="govuk-table__head">
 		<tr class="govuk-table__row">
 			<th class="govuk-table__header" scope="col">Description</th>
-			<th class="govuk-table__header" scope="col">Control List Entry</th>
+			<th class="govuk-table__header" scope="col">Control list entry</th>
```
Comment (andytaylorkainos) [inline templates/case/views/open-goods.html]: you missed one 👀

File: templates/case/open-licence-case.html (hunk 38 above)
Comment (andytaylorkainos) [inline templates/case/open-licence-case.html]: lots of strings in this file 👀

File: templates/case/open-licence-case.html (hunk 26 above)
Comment (andytaylorkainos) [reply templates/case/open-licence-case.html]: 👀

File: templates/case/views/standard-goods.html [hunk 41]
```diff
@@ -69,3 +60,4 @@ <h3 class="govuk-heading-m">Goods</h3>
 		{% endfor %}
 	</tbody>
 </table>
+<p class="govuk-body govuk-!-margin-top-3 govuk-!-margin-bottom-0 govuk-!-font-weight-bold">Total value: £{{ total_goods_value|intcomma }}</p>
```
Comment (markm5A8Z9F43913A) [inline templates/case/views/standard-goods.html]: Strings

File: templates/case/views/standard-goods.html [hunk 42]
```diff
@@ -26,41 +19,39 @@ <h3 class="govuk-heading-m">Goods</h3>
...
                 <td class="govuk-table__cell">
-                    {% if good.good.flags %}
-                        {% for flag in good.good.flags %}
-                            {{ flag.name }}<br>
-                        {% endfor %}
-                    {% endif %}
+                    {% for flag in good.good.flags %}
+                        {{ flag.name }}<br>
+					{% empty %}
+						None
```
Comment (markm5A8Z9F43913A) [inline templates/case/views/standard-goods.html]: Strings

File: templates/case/open-licence-case.html [hunk 43]
```diff
@@ -0,0 +1,281 @@
+{% extends 'layouts/base.html' %}
+
+{% load static svg %}
+
+{% block back_link %}
+	<div class="govuk-breadcrumbs">
+		<ol class="govuk-breadcrumbs__list">
+			<li class="govuk-breadcrumbs__list-item">
+				<a class="govuk-breadcrumbs__link" href="{% url 'cases:cases' %}?queue={{ queue.queue.id }}">{% if queue_name %}{{ queue_name }}{% else %}All cases{% endif %}</a>
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/views/open-goods.html (hunk 40 above)
Comment (jfaracik) [reply templates/case/views/open-goods.html]: Mate no more

File: templates/case/open-licence-case.html [hunk 44]
```diff
@@ -0,0 +1,273 @@
...
+											{% endfor %}
+										</div>
+									</td>
+								</tr>
+								{% endif %}
+								<tr class="lite-accordian-table__fold">
+									<td class="lite-accordian-table__fold-wrapper" colspan="7">
+										<div class="lite-accordian-table__fold-content">
+											<p class="govuk-body">
+												Destinations:
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/open-licence-case.html [hunk 45]
```diff
@@ -0,0 +1,273 @@
...
+				{% include 'case/includes/other-documents.html' %}
+			</div>
+		</div>
+
+		<!-- Activity -->
+		<div class="govuk-accordion__section">
+			<div class="govuk-accordion__section-header">
+				<h2 class="govuk-accordion__section-heading">
+					<span class="govuk-accordion__section-button" id="accordion-default-heading-3">
+						Activity
```
Comment (markm5A8Z9F43913A) [inline templates/case/open-licence-case.html]: Strings

File: templates/case/standard-licence-case.html [hunk 46]
```diff
@@ -0,0 +1,301 @@
...
+						{% lcs 'CASE_GOODS_LOCATION' %}
+					</span>
+				</h2>
+			</div>
+			<div id="accordion-goods-content" class="govuk-accordion__section-content" aria-labelledby="accordion-goods">
+				<table class="govuk-table">
+					<thead class="govuk-table__head">
+						<tr class="govuk-table__row">
+							<th scope="col" class="govuk-table__header govuk-!-width-one-third">Name</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-two-thirds">Address</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/standard-licence-case.html]: Strings

File: templates/case/standard-licence-case.html [hunk 47]
```diff
@@ -0,0 +1,301 @@
...
+				<table class="govuk-table" id="table-entities">
+					<thead class="govuk-table__head">
+						<tr class="govuk-table__row">
+							<th scope="col" class="govuk-table__header"></th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-third">Name</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Address</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Type</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Website</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Document</th>
+						</tr>
```
Comment (markm5A8Z9F43913A) [inline templates/case/standard-licence-case.html]: Strings

File: templates/case/standard-licence-case.html [hunk 48]
```diff
@@ -0,0 +1,301 @@
...
+					</thead>
+					<tbody class="govuk-table__body">
+						<tr class="govuk-table__row">
+							<th scope="row" class="govuk-table__header">Licencee</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/standard-licence-case.html]: Strings

File: templates/case/standard-licence-case.html [hunk 49]
```diff
@@ -184,16 +184,16 @@ <h2 class="govuk-accordion__section-heading">
...
-							<th scope="col" class="govuk-table__header govuk-!-width-one-third">Name</th>
-							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Address</th>
-							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Type</th>
-							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Website</th>
-							<th scope="col" class="govuk-table__header govuk-!-width-one-half">Document</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-third">CASE_PARTIES_NAME</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">CASE_PARTIES_ADDRESS</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">CASE_PARTIES_TYPE</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">CASE_PARTIES_WEBSITE</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-half">CASE_PARTIES_DOCUMENT</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/standard-licence-case.html]: Are you sure this works?

File: templates/case/standard-licence-case.html [hunk 50]
```diff
@@ -154,8 +154,8 @@ <h2 class="govuk-accordion__section-heading">
 				<table class="govuk-table">
 					<thead class="govuk-table__head">
 						<tr class="govuk-table__row">
-							<th scope="col" class="govuk-table__header govuk-!-width-one-third">Name</th>
-							<th scope="col" class="govuk-table__header govuk-!-width-two-thirds">Address</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-one-third">CASE_GOODS_LOCATION_NAME</th>
+							<th scope="col" class="govuk-table__header govuk-!-width-two-thirds">CASE_GOODS_LOCATION_ADDRESS</th>
```
Comment (markm5A8Z9F43913A) [inline templates/case/standard-licence-case.html]: This too.

**Pull request merged**